src/app/subscriptions-page/subscriptions-page.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+
+import { BehaviorSubject, combineLatestWith, Observable, shareReplay, Subscription as rxjsSubscription } from 'rxjs';
+import { map, switchMap, take, tap } from 'rxjs/operators';
+
+import { Subscription } from '../shared/subscriptions/models/subscription.model';
+import { buildPaginatedList, PaginatedList } from '../core/data/paginated-list.model';
+import { SubscriptionsDataService } from '../shared/subscriptions/subscriptions-data.service';
+import { PaginationComponentOptions } from '../shared/pagination/pagination-component-options.model';
+import { PaginationService } from '../core/pagination/pagination.service';
+import { PageInfo } from '../core/shared/page-info.model';
+import { AuthService } from '../core/auth/auth.service';
+import { EPerson } from '../core/eperson/models/eperson.model';
+import { getAllCompletedRemoteData } from '../core/shared/operators';
+import { RemoteData } from '../core/data/remote-data';
+import { hasValue } from '../shared/empty.util';
+
+@Component({
+  selector: 'ds-subscriptions-page',
+  templateUrl: './subscriptions-page.component.html',
+  styleUrls: ['./subscriptions-page.component.scss']
+})
+export class SubscriptionsPageComponent implements OnInit, OnDestroy {
```

Comment:
Again, please add description or typedocs to describe where this component is used